packages/stack/storage/package.json

```
@@ -40,7 +40,8 @@
     "lint": "eslint src/",
```

```suggestion
    "lint": "eslint src/ test/",
```

In #2285, among other changes, everything gets linted including tests.